packages/core/upload/admin/src/components/AssetTableList/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { BaseCheckbox } from '@strapi/design-system/BaseCheckbox';
+import { Table, Th, Thead, Tr } from '@strapi/design-system/Table';
+import { Typography } from '@strapi/design-system/Typography';
+import { VisuallyHidden } from '@strapi/design-system/VisuallyHidden';
+
+import { getTrad } from '../../utils';
+import { AssetDefinition } from '../../constants';
+import { tableHeaders } from './utils/tableHeaders';
+import { TableRows } from './TableRows';
+
+export const AssetTableList = ({ assets, onEditAsset, onSelectAsset, selectedAssets }) => {
+  const { formatMessage } = useIntl();
+
+  const headers = tableHeaders.map((header) => ({
```

Comment:
I don't think we need to do this here? We could do it on L45 inline and avoid the prior computation.